fastlane/lanes/release.rb

```diff
@@ -262,18 +332,40 @@ def trigger_buildkite_release_build(branch:, beta:)
   buildkite_annotate(style: 'info', context: 'trigger-release-build', message: message)
 end
 
-def create_release_backmerge_pr(version_to_merge:, next_version:)
+def create_backmerge_pr!
```

Comment:
Given an error is thrown in an alternative flow (isn't guaranteed to always happen), does it still make sense to add a `!` to the method name?